ipmininet/tests/test_iptables.py

```diff
@@ -0,0 +1,26 @@
+"""This module tests iptables"""
+from ipmininet.clean import cleanup
+from ipmininet.examples.iptables import IPTablesTopo
+from ipmininet.ipnet import IPNet
+from . import require_root
+
+
+@require_root
+def test_iptables_example():
+    try:
+        net = IPNet(topo=IPTablesTopo())
+        net.start()
+
+        ip = net["r2"].intf("r2-eth0").ip
+        cmd = "ping -W 1 -c 1 %s" % ip
+        p = net["r1"].popen(cmd.split(" "))
+        assert p.wait() == 0, "pings over IPv4 should not be blocked"
+
+        ip6 = net["r2"].intf("r2-eth0").ip6
+        cmd = "ping6 -W 1 -c 1 %s" % ip6
+        p = net["r1"].popen(cmd.split(" "))
+        assert p.wait() != 0, "pings over IPv6 should be blocked"
+
```

Comment:
Could reverse tests be also added, i.e., to show that _some_ connectivity works (and raise errors if it does not)